project/GoogleDriveV3.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import print_function
+from apiclient.discovery import build
+from httplib2 import Http
+from oauth2client import file, client, tools
+from oauth2client import clientsecrets
+import googleapiclient.errors
+
+#this module uses Google Drive v3 API because there is no option to
+# work with folders in Google Spreadsheet v4 API
+
+def buildService(file):
+
+    # Setup the Drive v3 API
+    SCOPES = 'https://www.googleapis.com/auth/drive'
+    store = file.Storage('credentials.json')
+    creds = store.get()
+    if not creds or creds.invalid:
+        try:
+            flow = client.flow_from_clientsecrets('client_secret.json', SCOPES)
+            creds = tools.run_flow(flow, store)
+        except clientsecrets.InvalidClientSecretsError:
+            print('The client secrets were missing or invalid: ')
+        except client.UnknownClientSecretsFlowError:
+            print('This OAuth 2.0 flow is unsupported')
+        except client.Error:
+            print('Unexpected Error')
+
+    service = build('drive', 'v3', http=creds.authorize(Http()))
+
+    return service
+
+def moveFileToFolder(fileID, folderID):
+
+    global file
+
+    service = buildService(file)
```

Comment:
We will want to error check that the service is valid in the action function and error out accordingly